packages/dropdowns/src/styled/field/StyledSelect.ts

```diff
@@ -66,17 +63,20 @@ export interface IStyledSelectProps {
   validation?: VALIDATION;
 }
 
-const StyledSelect = styled(FauxInput).attrs<IStyledSelectProps>(props => ({
+export const StyledSelect = styled(FauxInput).attrs<IStyledSelectProps>(props => ({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION,
   'aria-invalid': isInvalid(props.validation)
 }))<IStyledSelectProps>`
   position: relative;
   cursor: ${props => (props.disabled ? 'default' : 'pointer')};
   appearance: none;
+  overflow: hidden;
   /* stylelint-disable-next-line property-no-unknown */
   padding-${props => (props.theme.rtl ? 'left' : 'right')}: ${props => getIconSize(props)};
   text-align: ${props => props.theme.rtl && 'right'};
+  text-overflow: ellipsis;
+  white-space: nowrap;
```

Comment:
This is not a 100% effective approach for this problem. Overflow treatment is lost at extreme sizes...

![Screen Shot 2020-02-10 at 10 20 32 AM](https://user-images.githubusercontent.com/143773/74177649-074d6100-4bef-11ea-82c6-73e16cc70e51.png)

Remove all of these. Instead, add a `<div style={{ overflow: 'hidden', textOverflow: 'ellipsis' }}>{children}</div>` (or Styled component) that surrounds the children of `Select` and `Autocomplete`.

Reply:
This is now moved to a standalone styled element which wraps the text provided.